fcli-core/fcli-aviator-common/src/main/java/com/fortify/cli/aviator/core/AuditFPR.java

```diff
@@ -22,7 +22,7 @@
 public class AuditFPR {
     private static final Logger LOG = LoggerFactory.getLogger(AuditFPR.class);
 
-    public static File auditFPR(File file, String token, String url, String appVersion,IAviatorLogger logger) throws AviatorSimpleException, AviatorTechnicalException, IOException {
+    public static File auditFPR(File file, String token, String url, String appVersion,IAviatorLogger logger) throws Exception {
```

Comment:
Why did you change exception handling here? In general, it's bad practice to have `throws Exception`; better to explicitly list the exception types being thrown, or probably even better, translate into an `AviatorTechnicalException` as you were doing before. Side note; why does `AviatorTechnicalException` extend from `Exception` instead of `RuntimeException? The latter would avoid having to explicitly catch or declare `AviatorTechnicalException`, and is also what we do for Fcli exceptions.